modules/neovim/hm.nix

```diff
@@ -2,9 +2,13 @@
 
 {
   options.stylix.targets.neovim = {
+    colorscheme = lib.mkOption {
```

Comment:
Could we rename this option to something like `stylix.targets.neovim.plugin`? `colorscheme` feels very generic within a project which is all about colorschemes.